src/components/AuthContainer.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <form class="auth-container">
+    <img src="../assets/logo-wolox.png" class="logo" />
+    <slot name="content"></slot>
+    <button
+      type="button"
```

Comment:
Change to `submit` since it's this button's type. Also, add the `@submit.prevent` to the `form` tag